<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1997</width>
    <height>1120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>840</y>
      <width>41</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>411</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="titleLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="authorLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="iDLabel">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="genreLabel">
        <property name="text">
         <string>Genre</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="genreLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="issueNoLabel">
        <property name="text">
         <string>Issue No.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="issueLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="addItemButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>891</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="libraryTableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>811</width>
       <height>481</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Author</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Genre/Issue Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Borrowed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Borrower Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Borrow Date</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="toggleBorrowedButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Borrowed Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeItemButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove selected item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="borrowReturnButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrow/Return</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1997</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
